code/game/turfs/simulated/floor/misc_floor.dm

```diff
@@ -265,3 +265,36 @@
 	for(var/obj/structure/spacevine/SV in src)
 		if(!QDESTROYING(SV))//Helps avoid recursive loops
 			qdel(SV)
+
+/turf/open/floor/plasteel/acid_spray
+	floor_tile = /obj/item/stack/tile/acid_spray
+	var/spray_amount = 35
+	var/cooldown = 150
+	var/last_spray = 0
+	var/primed = FALSE
+	
+/turf/open/floor/plasteel/acid_spray/New()
+	..()
+	last_spray = world.time
+	primed = TRUE //prevents firing immediately when placed
```

Comment:
Without it, the tile sprays as soon as it's placed if there's something on the turf.